controllers/autohealconfig_controller.go

```diff
@@ -18,13 +18,25 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
 
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
-	ctrl "sigs.k8s.io/controller-runtime"
+	"k8s.io/apimachinery/pkg/util/yaml"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
-	autohealv1alpha1 "github.com/bf2fc6cc711aee1a0c2a/autoheal-operator/api/v1alpha1"
+	v1alpha1 "github.com/bf2fc6cc711aee1a0c2a/autoheal-operator/api/v1alpha1"
```

Comment:
```suggestion
	"github.com/bf2fc6cc711aee1a0c2a/autoheal-operator/api/v1alpha1"
```
Redundant alias as it's already `v1alpha1`.